assets/js/src/core/modules/element/editor/shared-tab-manager/tabs/schedule/schedule-container.tsx

```diff
@@ -74,14 +76,15 @@ export const ScheduleTabContainer = (): React.JSX.Element => {
     }
   }, [schedules])
 
-  if (isLoading || data === undefined) {
-    return <Content loading />
-  }
-
   if (isError) {
+    trackError(new ApiError(error))
     return <div>Error</div>
   }
 
+  if (isLoading || data === undefined) {
+    return <Content loading />
+  }
+
```

Comment:
@markus-moser, as you see here, I moved the `isError` condition above the `isLoading` condition since rtk doesn't reset the `isLoading` when an error is thrown. 

This leads to the behavior that now "error" appears in the tab instead of an infinite loading spinner ... Should we define an "error view" simmilar to the empty content or loading view?

WDYT

Reply:
Yes I would go for an error view where we show the error message instead - maybe a simple <Alert error...> (and then remove the default handling with the alert).

Reply:
done with: https://github.com/pimcore/studio-ui-bundle/pull/1014/commits/ce10e6551fc620eb4aaa339a279787df44aae2f4